README.md

```diff
@@ -20,7 +20,17 @@ For convenience, you may set it as a global default option to all `KingfisherMan
 
 ```swift
 // somewhere after your application launches...
-KingfisherManager.shared.defaultOptions = [.processor(WebPProcessor.default), .cacheSerializer(WebPSerializer.default)]
+let modifier = AnyModifier { request in
+  var req = request
+  req.addValue("image/webp */*", forHTTPHeaderField: "Accept")
+  return req
+}
+
+KingfisherManager.shared.defaultOptions += [
+  .requestModifier(modifier),
```

Comment:
It seems the modifier is not always required for this library to work, so would it be better to separate it from the other options and document it out, like this?
```swift
Kingfisher.shared.defaultOptions += [
    .processor(WebPProcessor.default),
    .cacheSerializer(WebPSerializer.default)
]

// Some services expect "image/webp" in header to return a webp image.
let modifier = AnyModifier { request in
    var req = request
    req.addValue("image/webp */*", forHTTPHeaderField: "Accept")
    return req
}
KingfisherManager.shared.defaultOptions += [
    .requestModifier(modifier)
]
```